plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/OpenShiftServerEditorSection.java

```diff
@@ -616,7 +594,7 @@ private Job loadingProjectsJob(OpenShiftServerEditorModel model, IServerWorkingC
 	private void loadResources(final Composite container, OpenShiftServerEditorModel model) {
 		IServerWorkingCopy server = input.getServer();
 
-		final Connection connection = OpenShiftServerUtils.getConnection(server);
+		OpenShiftServerUtils.getConnection(server);
```

Comment:
I can't see any side-effect at all. Seems to be a 100% static getter. Doesn't seem necessary at all. Removing it.